mobile/src/pages/ImageCreate.tsx

```diff
@@ -1,56 +1,100 @@
+import { StatusBar } from "expo-status-bar";
 import { useState } from "react";
-import { Text, Pressable, View, ScrollView, SafeAreaView } from "react-native";
+import { useMutation, useQuery } from "@tanstack/react-query";
+import { ActivityIndicator, Text, Pressable, View, ScrollView, SafeAreaView } from "react-native";
 import { imageStyles } from "../data/styles";
 import Button from "../components/Button";
 import Card from "../components/Card";
 import Icon from "react-native-vector-icons/Ionicons";
+import MultiSelect from "../components/MultiSelect";
 import BottomNavBar from "../components/BottomNavBar";
 import TopNavBar from "../components/TopNavBar";
-import SelectableChip from "../components/SelectableChip";
```

Comment:
Don't forget to delete this file in the next PR since it's not going to be used.